### ISSUE TITLE
Add examples for common ADA violations in README

### ISSUE THREAD

**Issue by Ethan-Kruger** [Ethan-Kruger/ADA-Checker#36]
Expand the README with concrete examples of common ADA accessibility violations to help users and contributors recognize and address issues.
